templates/control

```diff
@@ -3,6 +3,6 @@ Version: {{ node_deb_package_version }}
 Section: base
 Priority: optional
 Architecture: all
-Depends: nodejs
+Depends: nodejs , {{ node_deb_package_dependencies }}
```

Comment:
sure i think the points you made are valid will make those changes and update. And `node_deb_package_additional_dependencies` is too long for command line usage. May be `node_deb_system_dependencies`, But if you feel `node_deb_package_additional_dependencies` is more clear i am fine with it.